### PR TITLE
update instructions on running the broker

### PR DESCRIPTION
**Pull request by nimakaviani**
- add services directory as an input arg to the binary
- add services as a volume to docker

 - [x ] document your feature
 - [x ] comment your code
 - [ x] update the README / how-to-run

### CODE REVIEW

**Comment by MHBauer** [comment]
I think this should all be rolled into #27

**Comment by nimakaviani** [comment]
thought so too. but that PR is becoming really  big. we can hold back on this and  once #27 is merged, we can rebase this on master before merging.

File: Dockerfile [hunk 1]
```diff
@@ -6,5 +6,4 @@ run go build -v -o /broker ./cmd/broker
 from alpine:3.8
 expose 3333
 copy --from=builder /broker /broker
-add ./creds.json /creds.json
-cmd ["/broker","/creds.json"]
+cmd ["/broker","/config.json", "/services"]
```
Comment (MHBauer) [inline Dockerfile]: need this

File: README.md [hunk 2]
```diff
@@ -7,8 +7,8 @@ Configure the broker, by setting `username` and `password` in `creds.json`.
 
 Run the broker by doing:
 
-    go build cmd/broker && ./broker creds.json
+    go build ./cmd/broker && ./broker config.json services/
 
 OR
 
-    docker build -t blockhead . && docker run -v $(pwd)/config.json:/config.json -p 3333:3333 blockhead
+    docker build -t blockhead . && docker run -v $(pwd)/config.json:/config.json -v $(pwd)/services:/services -p 3333:3333 blockhead
```
Comment (MHBauer) [inline README.md]: but I would drop this

File: config.json [hunk 3]
```diff
@@ -1,5 +1,4 @@
 {
-    "username":"",
-    "password":"",
-    "port": 3333
+    "Username":"",
+    "Password":""
```
Comment (swetharepakula) [inline config.json]: I think this is an artifact of the previous branch.

**Comment by swetharepakula** [comment]
Does this PR make sense anymore then? since the dockerfile still needs docker to work?

**Comment by nimakaviani** [comment]
yeah on the fence here. let's keep it around but we dont need to merge it for now.

File: Dockerfile [hunk 4]
```diff
@@ -7,4 +7,6 @@ from alpine:3.8
 expose 3333
 copy --from=builder /broker /broker
 add ./config.json /config.json
-cmd ["/broker","/config.json"]
+run mkdir -p /services
+copy services/* /services
```
Comment (MHBauer) [inline Dockerfile]: I think this can be done in one step with only the copy and without the "run"

Comment (nimakaviani) [reply Dockerfile]: I vaguely remember trying only the `copy` and that didn't work.  worth giving it another try.

**Comment by MHBauer** [comment]
confused the heck out of myself, but I think this is what we want to do?

**Comment by nimakaviani** [comment]
ideally we want the docker container to have `solc` and `node` running on it too.

**Comment by nimakaviani** [comment]
sounds good. let's fix it when we do another pass on this file.

**Pull request merged**